_posts/en/newsletters/2025-02-21-newsletter.md

```diff
@@ -0,0 +1,282 @@
+---
+title: 'Bitcoin Optech Newsletter #342'
+permalink: /en/newsletters/2025/02/21/
+name: 2025-02-21-newsletter
+slug: 2025-02-21-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes an ideal for allowing mobile wallets to
```

Comment:
Is this a typo or it was meant like this?

```suggestion
This week's newsletter describes an idea for allowing mobile wallets to
```

Reply:
Typo, thanks!  Fixing.